feat(howitworks.jsx): designing a section component

Bolphunga opened the pull request:
Changes made:
Added a section explaining how the website works.


Screenshot:


Related issue:
Resolves #21 

# Related Issue

- Resolve #21

samiraTbl committed:
feat(howitworks.jsx): designing a section component

Design features section component #21

0m3ga13 commented on src/components/sections/HowItWorks.jsx:
```diff
@@ -0,0 +1,30 @@
+const Hiw = () => {
```
I sugegest changing the name of this component for something more representative

medshk reviewed (commented):
you can name it AppFunctionalityDescription

samiraTbl committed:
style(appfunctionalitydescription): i changed the name of the file component

i changed the name of the componenet from hiw to AppFunctionalityDescription

designing a section component #33

0m3ga13 commented:
well done Samira!

Bolphunga committed:
feat(howitworks): added how it works section

Bolphunga committed:
Merge branch 'develop' into 21-design-features-section-component

Emybel reviewed (approved):
Great job Samira!
